### PR TITLE
(GH-556) Use OutputDirectory when creating Chocolatey Package

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

**Comment by gep13** [comment]
@devlead @patriksvensson can one of you have a quick look at this?  Thanks!

**Comment by patriksvensson** [comment]
@gep13 Sure :smile:

File: build.cake [hunk 1]
```diff
@@ -162,6 +162,7 @@ Task("Create-Chocolatey-Packages")
   ChocolateyPack("./nuspec/Cake.Portable.nuspec", new ChocolateyPackSettings {
       Version = semVersion,
 	  ReleaseNotes = releaseNotes.Notes.ToArray(),
```
Comment (patriksvensson) [inline build.cake]: Could you also fix spacing here while you're at it?

**Comment by patriksvensson** [comment]
Looks good to me :+1: 
Left a minor comment. If you want to fix that it would be great.
Tell me when you're ready and I'll merge it.

**Comment by gep13** [comment]
Ok, fixed up, although, turns out all the lines were incorrectly tabbed.  The one that was "wrong" was actually correct :-D

**Comment by patriksvensson** [comment]
OK, will merge as soon as CI pass.

**Comment by patriksvensson** [comment]
Merged :+1:

**Pull request merged**